sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantRenameSitePrioritization.md

```diff
@@ -0,0 +1,73 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/set-spotenantrenamesiteprioritization
+applicable: SharePoint Online
+title: Set-SPOTenantRenameSitePrioritization
+author: AnfraMsft
+ms.author: anfra
+ms.reviewer: anushmag
+manager: anushmag
+schema: 2.0.0
+ms.topic: reference
+---
+
+# Set-SPOTenantRenameSitePrioritization
+
+## SYNOPSIS
+This cmdlet allows prioritization of a site for early execution, as part of [Advanced Tenant Rename](https://aka.ms/advancedtenantrename).
```

Comment:
>https://aka.ms/advancedtenantrename [](http://example.com/codeflow?start=101&length=35)

replace this aka.ms with the relative URL `/sharepoint/change-your-sharepoint-domain-name#advanced-tenant-rename-preview`

Reply:
applies to this  place and others